NuKeeper.Tests/Engine/ForkFinderTests.cs

```diff
@@ -30,7 +29,7 @@ public async Task ThrowsWhenNoPushableForkCanBeFound()
         public async Task FallbackForkIsUsedWhenItIsFound()
         {
             var fallbackFork = DefaultFork();
-            var fallbackRepoData = RepositoryBuilder.MakeRepository();
+            var fallbackRepoData = new GitHubRepository(RepositoryBuilder.MakeRepository());
```

Comment:
We should split up the tests in the future as well. That will be to much for now probably